src/views/dashboard/panel/Tool.vue

```diff
@@ -585,6 +604,19 @@ limitations under the License. -->
     }
     return resp;
   }
+  async function updateCurrentDestEndpoint() {
+    const resp = await selectorStore.getEndpoint(params.destPodId, true);
+
+    if (resp.errors) {
+      return ElMessage.error(resp.errors);
+    }
+    const pod = resp.data.endpoint || {};
+    if (!pod.id) {
+      ElMessage.info("The Destination endpoint ID doesn't exist.");
```

Comment:
`Destination`? Is this related to Endpoint Relation?

Reply:
How about source?

Reply:
> `Destination`? Is this related to Endpoint Relation?

Yes

Reply:
> How about source?

it is named `endpoint` or `instance`.